core/storage/exploration/gae_models_test.py

```diff
@@ -64,6 +64,21 @@ def test_get_exploration_count(self):
         self.assertEqual(saved_exploration.category, 'A Category')
         self.assertEqual(saved_exploration.objective, 'An Objective')
 
+    def test_reconstitute(self):
+        exploration = exp_domain.Exploration.create_default_exploration(
+            'id', title='A Title',
+            category='A Category', objective='An Objective')
+        exp_services.save_new_exploration('id', exploration)
+        exp_model = exp_models.ExplorationModel.get_by_id('id')
+        snapshot_dict = exp_model.compute_snapshot()
+        snapshot_dict['skill_tags'] = ['tag1', 'tag2']
+        snapshot_dict['default_skin'] = 'conversation_v1'
+        snapshot_dict['skin_customizations'] = {}
+        exp_model = exp_model._reconstitute(snapshot_dict) # pylint: disable=protected-access
```

Comment:
Try to not add any pylint disables if it is not absolutely needed, here you can use `get_version` or you can look how it is handled in `ExplorationRightsModel` (the use of `convert_to_valid_dict`)